client/src/lib.rs

```diff
@@ -714,9 +776,22 @@ pub trait WorkflowClientTrait {
 /// Optional fields supplied at the start of workflow execution
 #[derive(Debug, Clone, Default)]
 pub struct WorkflowOptions {
+    /// Set the policy for reusing the workflow id
+    pub id_reuse_policy: WorkflowIdReusePolicy,
+
+    /// Optionally set the execution timeout for the workflow
+    /// https://docs.temporal.io/workflows/#workflow-execution-timeout
```

Comment:
CI is complaining about auto-link. You can use `cargo doc --workspace --all-features --no-deps` to see doc issues

Reply:
fixed